services/search/tests/routes/test_filter.py

```diff
@@ -12,12 +12,12 @@
 import pytest
 from datasets import Dataset
 from libapi.exceptions import InvalidParameterError
-from libapi.response import ROW_IDX_COLUMN, create_response
+from libapi.response import create_response
 from libcommon.storage_client import StorageClient
 
 from search.config import AppConfig
 from search.routes.filter import execute_filter_query, validate_query_parameter
-
+from libcommon.constants import ROW_IDX_COLUMN
```

Comment:
Can you run `make install; make style` in `services/search`, to fix the linting issue (we automatically order the imports)? I can do it for you if you prefer.

Reply:
It's what makes the CI fail:

https://github.com/huggingface/dataset-viewer/actions/runs/9106562973/job/25035748026?pr=2810#step:8:16

```
 src/search/routes/search.py:4:1: I001 [*] Import block is un-sorted or un-formatted
Found 1 error.
[*] 1 fixable with the `--fix` option.
```

Reply:
> Can you run `make install; make style` in `services/search`, to fix the linting issue (we automatically order the imports)? I can do it for you if you prefer.

Would you please do that?